### PR TITLE
Make help more clear for #4232

### PR DESCRIPTION
**Pull request by deajan**

### CODE REVIEW

File: src/opnsense/mvc/app/library/OPNsense/Backup/Nextcloud.php [hunk 1]
```diff
@@ -54,8 +54,8 @@ public function getConfigurationFields()
             array(
                 "name" => "url",
                 "type" => "text",
-                "label" => gettext("URL"),
-                "help" => gettext("The Base URL to Nextcloud without trailing slash. For example: https://cloud.example.com"),
+                "label" => gettext("Nextcloud Server URL"),
+                "help" => gettext("The Base URL to Nextcloud without trailing slash. For example: https://cloud.example.com If your server SSL certificate is not issued from a well known authority, you will need to add your intermediate certificate to the OPNSense CA truststore to avoid SSL verification errors."),
```
Comment (fabianfrz) [inline src/opnsense/mvc/app/library/OPNsense/Backup/Nextcloud.php:58]: This one is basic network knowlege and expected from users of OPNsense to know (that certificates are validated and how). I would not like that but I don't care if it is merged as well.

File: src/opnsense/mvc/app/library/OPNsense/Backup/Nextcloud.php [hunk 2]
```diff
@@ -69,7 +69,7 @@ public function getConfigurationFields()
                 "name" => "password",
                 "type" => "password",
                 "label" => gettext("Password"),
-                "help" => gettext("The app password which has been generated for you"),
+                "help" => gettext("The app password which has been generated for you in Nextcloud security settings. Do not use the real password here."),
```
Comment (fabianfrz) [inline src/opnsense/mvc/app/library/OPNsense/Backup/Nextcloud.php:72]: :+1:

File: src/opnsense/mvc/app/library/OPNsense/Backup/Nextcloud.php (hunk 1 above)
Comment (rawtaz) [reply src/opnsense/mvc/app/library/OPNsense/Backup/Nextcloud.php:58]: There's a (no more) surprising number of people having no clue about those things though, so IMO it's better to have it there than not. Saves some grey hairs ;)

Comment (fichtner) [reply src/opnsense/mvc/app/library/OPNsense/Backup/Nextcloud.php:58]: Again here: documentation can be added and expended. We do not need to overdo GUI labels and help texts.

**Comment by fichtner** [comment]
In the meantime nextcloud backup module moved to plugins.

**Pull request closed without merging**